custom/icds_reports/sqldata/exports/growth.py

```diff
@@ -0,0 +1,179 @@
+from corehq.apps.locations.models import SQLLocation
+from custom.icds_reports.models import ChildHealthMonthlyView
+from custom.icds_reports.sqldata.base import IcdsSqlData
+from custom.icds_reports.utils.mixins import ExportableMixin
+from custom.icds_reports.utils import get_status, calculate_date_for_age, \
+    current_month_stunting_column, \
+    current_month_wasting_column, phone_number_function, india_now
+from dateutil.relativedelta import relativedelta
+
+
+class GrowthExport(ExportableMixin, IcdsSqlData):
+    title = 'Child Growth Tracking'
+    table_name = 'child_health_monthly_view'
+
+    def get_excel_data(self, location):
+
+        def _check_case_presence(case_id, column, data_dict):
+            return data_dict[case_id][column] if case_id in data_dict.keys() else "N/A"
+
+        filters = {
+            "month": self.config['month'],
+            "age_in_months__lte": 60,
```

Comment:
does this spec ask to put this limit explicitly because indicators in this report, PSE and LUNCH are given from 3 to 6 years.

Reply:
for Both PSE and Lunch there is a lower bound (not sure if there is a upper bound or not) that's why keep this

Reply:
Upper bound should not be 60. This report would include Children 5-6 yrs as well

Reply:
It's outdated, it's updated to 72